lms/envs/production.py

```diff
@@ -561,9 +561,13 @@ def get_env_setting(setting):
 
 # Event tracking
 TRACKING_BACKENDS.update(AUTH_TOKENS.get("TRACKING_BACKENDS", {}))
-EVENT_TRACKING_BACKENDS['tracking_logs']['OPTIONS']['backends'].update(AUTH_TOKENS.get("EVENT_TRACKING_BACKENDS", {}))
 EVENT_TRACKING_BACKENDS['segmentio']['OPTIONS']['processors'][0]['OPTIONS']['whitelist'].extend(
     AUTH_TOKENS.get("EVENT_TRACKING_SEGMENTIO_EMIT_WHITELIST", []))
+
+ENV_EVENT_TRACKING_BACKENDS = AUTH_TOKENS.get("EVENT_TRACKING_BACKENDS", {})
```

Comment:
I like that defensive logic with the default empty dir. This also doesn't make a problem in iter in the for.